pytorch_lightning/strategies/launchers/subprocess_script_hydra.py

```diff
@@ -0,0 +1,82 @@
+import os
+from typing import List, Optional, Tuple
+
+import __main__
+import torch
+
+from pytorch_lightning.strategies.launchers.subprocess_script import _SubprocessScriptLauncher
+from pytorch_lightning.utilities import _HYDRA_AVAILABLE
+
+if _HYDRA_AVAILABLE:
+    from hydra.core.hydra_config import HydraConfig
+    from hydra.utils import to_absolute_path
+
+
+class _SubprocessScriptHydraLauncher(_SubprocessScriptLauncher):
+    """Hydra Launcher subclasses `_SubprocessScriptLauncher to manage Hydra specific commands when running under
+    Hydra environment."""
+
+    def _get_complete_path(self, command: str) -> str:
+        return to_absolute_path(command)
+
+    def _get_launch_command(self, command: List[str], local_rank: int) -> Tuple[List[str], Optional[str]]:
+        """Modifies the command to support Hydra initiated processes."""
+        if not HydraConfig.initialized():
+            return command, None
+
+        # If Hydra is initialized:
+        #   1) Set `cwd` to the hydra working directory
+        #   2) Use the stored configuration in `hydra_cfg.output_subdir / config.yaml` to spawn a new child
+
+        cwd = os.getcwd()
```

Comment:
Does the `cwd` need to be part of this?

AFAIK neither of the implementations do anything special with it.

Isn't returning `None` equal to returning `os.getcwd()`?

At least, in the previous code, this was `hydra.utils.get_original_cwd()`, not sure if this has been removed by mistake.

Reply:
I noticed, this too. This was done before I integrated the launcher, so need to confirm with @jgbos .

Reply:
much delayed, but you are correct on this